site/profile/manifests/app/tse_master/files/splunk.pp

```diff
@@ -1,88 +1,88 @@
-class profile::app::tse_master::files::splunk (
```

Comment:
This probably shouldn't be in the PR either right? Would it better to just delete this file?